Package.swift

```diff
@@ -45,15 +45,15 @@ let package = Package(
         .target(
             /** Cross-platform access to bare `libc` functionality. */
             name: "TSCLibc",
-            dependencies: ["TSCclibc"]),
+            dependencies: []),
         .target(
             /** TSCBasic support library */
             name: "TSCBasic",
-            dependencies: ["TSCLibc"]),
+            dependencies: ["TSCLibc", "TSCclibc"]),
```

Comment:
Hmm, here you have added a dependency on `TSCclibc` for `TSCBasic`, but I don't see that on the CMake side, is that omission intentional?  Is this working around a bug?

Reply:
Not intentional.  This is because it's needed for some code in Process.swift (which is new since this patch was first written), so it does need to be a dependency.

I was puzzled why I didn't see this in the testing I did using SwiftPM's `bootstrap` script, which builds everything with CMake.  But I just realized that it's using the copy of ToolsSupportCore that's still vendored inside SwiftPM.  Now that I see that I wasn't properly testing these changes in `bootstrap`, I'll make sure it works and update this PR.

Thanks for catching this!

Reply:
Pushed a fix for this.